app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\UserUpdateRequest;
+use App\Models\User;
+use App\Services\UserService;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class UserController extends Controller
+{
+    public function __construct(UserService $userService)
+    {
+        $this->userService = $userService;
+    }
+
+    public function getUsers(): JsonResponse
+    {
+        $users = User::all();
+
+        return response()->json([
+            'users' => $users,
+            'message' => 'Users found',
```

Comment:
where do we use 'message' and 'status' values? According to the 'get users' task, only user emails are required to be returned. Btw 'users' must contain only users emails